[2024-03-18T01:31:48.435+0000] {processor.py:157} INFO - Started process (PID=173) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:31:48.436+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:31:48.439+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:31:48.438+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:32:18.451+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:32:18.450+0000] {timeout.py:68} ERROR - Process timed out, PID: 173
[2024-03-18T01:32:18.469+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:32:18.453+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_daily.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_daily.py", line 4, in <module>
    from scripts.main import data_ingestion
  File "/opt/airflow/scripts/main.py", line 423, in <module>
    data_ingestion()
  File "/opt/airflow/scripts/main.py", line 415, in data_ingestion
    df_ecobici_stations, df_ecobici_stations_status = get_ecobici_data(base_url, params)
  File "/opt/airflow/scripts/main.py", line 232, in get_ecobici_data
    r_ecobiciSI = requests.get(full_url_ecobiciSI, params=params)
  File "/home/airflow/.local/lib/python3.8/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/requests/sessions.py", line 747, in send
    r.content
  File "/home/airflow/.local/lib/python3.8/site-packages/requests/models.py", line 899, in content
    self._content = b"".join(self.iter_content(CONTENT_CHUNK_SIZE)) or b""
  File "/home/airflow/.local/lib/python3.8/site-packages/requests/models.py", line 816, in generate
    yield from self.raw.stream(chunk_size, decode_content=True)
  File "/home/airflow/.local/lib/python3.8/site-packages/urllib3/response.py", line 628, in stream
    data = self.read(amt=amt, decode_content=decode_content)
  File "/home/airflow/.local/lib/python3.8/site-packages/urllib3/response.py", line 567, in read
    data = self._fp_read(amt) if not fp_closed else b""
  File "/home/airflow/.local/lib/python3.8/site-packages/urllib3/response.py", line 533, in _fp_read
    return self._fp.read(amt) if amt is not None else self._fp.read()
  File "/usr/local/lib/python3.8/http/client.py", line 459, in read
    n = self.readinto(b)
  File "/usr/local/lib/python3.8/http/client.py", line 503, in readinto
    n = self.fp.readinto(b)
  File "/usr/local/lib/python3.8/socket.py", line 669, in readinto
    return self._sock.recv_into(b)
  File "/usr/local/lib/python3.8/ssl.py", line 1274, in recv_into
    return self.read(nbytes, buffer)
  File "/usr/local/lib/python3.8/ssl.py", line 1132, in read
    return self._sslobj.read(len, buffer)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/timeout.py", line 69, in handle_timeout
    raise AirflowTaskTimeout(self.error_message)
airflow.exceptions.AirflowTaskTimeout: DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 173
[2024-03-18T01:32:18.471+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:32:18.493+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/data_ingestion_daily.py took 30.061 seconds
[2024-03-18T01:32:28.604+0000] {processor.py:157} INFO - Started process (PID=180) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:32:28.606+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:32:28.608+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:32:28.607+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:33:00.766+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:33:00.765+0000] {timeout.py:68} ERROR - Process timed out, PID: 180
[2024-03-18T01:33:00.766+0000] {logging_mixin.py:154} INFO - No es posible conectar a Redshift
[2024-03-18T01:33:00.767+0000] {logging_mixin.py:154} INFO - DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 180
[2024-03-18T01:33:00.767+0000] {logging_mixin.py:154} INFO - No es posible crear tabla agencies en DB: 
[2024-03-18T01:33:00.768+0000] {logging_mixin.py:154} INFO - local variable 'conn' referenced before assignment
[2024-03-18T01:33:00.768+0000] {logging_mixin.py:154} INFO - No es posible crear tabla bus_positions en DB: 
[2024-03-18T01:33:00.769+0000] {logging_mixin.py:154} INFO - local variable 'conn' referenced before assignment
[2024-03-18T01:33:00.769+0000] {logging_mixin.py:154} INFO - No es posible crear tabla ecobici_stations en DB: 
[2024-03-18T01:33:00.770+0000] {logging_mixin.py:154} INFO - local variable 'conn' referenced before assignment
[2024-03-18T01:33:00.770+0000] {logging_mixin.py:154} INFO - No es posible crear tabla ecobici_stations_status en DB: 
[2024-03-18T01:33:00.770+0000] {logging_mixin.py:154} INFO - local variable 'conn' referenced before assignment
[2024-03-18T01:33:00.783+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:33:00.771+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_daily.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_daily.py", line 4, in <module>
    from scripts.main import data_ingestion
  File "/opt/airflow/scripts/main.py", line 423, in <module>
    data_ingestion()
  File "/opt/airflow/scripts/main.py", line 417, in data_ingestion
    db_keys, conn = connect_to_redshift()
  File "/opt/airflow/scripts/main.py", line 385, in connect_to_redshift
    return db_keys, conn
UnboundLocalError: local variable 'conn' referenced before assignment
[2024-03-18T01:33:00.785+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:33:00.818+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/data_ingestion_daily.py took 32.216 seconds
[2024-03-18T01:33:10.892+0000] {processor.py:157} INFO - Started process (PID=186) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:33:10.893+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:33:10.895+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:33:10.894+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:33:44.765+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:33:44.765+0000] {timeout.py:68} ERROR - Process timed out, PID: 186
[2024-03-18T01:33:44.766+0000] {logging_mixin.py:154} INFO - No es posible conectar a Redshift
[2024-03-18T01:33:44.767+0000] {logging_mixin.py:154} INFO - DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 186
[2024-03-18T01:33:44.768+0000] {logging_mixin.py:154} INFO - No es posible crear tabla agencies en DB: 
[2024-03-18T01:33:44.768+0000] {logging_mixin.py:154} INFO - local variable 'conn' referenced before assignment
[2024-03-18T01:33:44.769+0000] {logging_mixin.py:154} INFO - No es posible crear tabla bus_positions en DB: 
[2024-03-18T01:33:44.769+0000] {logging_mixin.py:154} INFO - local variable 'conn' referenced before assignment
[2024-03-18T01:33:44.769+0000] {logging_mixin.py:154} INFO - No es posible crear tabla ecobici_stations en DB: 
[2024-03-18T01:33:44.770+0000] {logging_mixin.py:154} INFO - local variable 'conn' referenced before assignment
[2024-03-18T01:33:44.771+0000] {logging_mixin.py:154} INFO - No es posible crear tabla ecobici_stations_status en DB: 
[2024-03-18T01:33:44.771+0000] {logging_mixin.py:154} INFO - local variable 'conn' referenced before assignment
[2024-03-18T01:33:44.781+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:33:44.772+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_daily.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_daily.py", line 4, in <module>
    from scripts.main import data_ingestion
  File "/opt/airflow/scripts/main.py", line 423, in <module>
    data_ingestion()
  File "/opt/airflow/scripts/main.py", line 417, in data_ingestion
    db_keys, conn = connect_to_redshift()
  File "/opt/airflow/scripts/main.py", line 385, in connect_to_redshift
    return db_keys, conn
UnboundLocalError: local variable 'conn' referenced before assignment
[2024-03-18T01:33:44.783+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:33:44.805+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/data_ingestion_daily.py took 33.915 seconds
[2024-03-18T01:33:54.919+0000] {processor.py:157} INFO - Started process (PID=192) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:33:54.920+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:33:54.922+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:33:54.921+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:34:15.138+0000] {logging_mixin.py:154} INFO - Conectado a Redshift con éxito!
[2024-03-18T01:34:16.555+0000] {logging_mixin.py:154} INFO - Tabla agencies creada con exito en DB
[2024-03-18T01:34:17.387+0000] {logging_mixin.py:154} INFO - Tabla bus_positions creada con exito en DB
[2024-03-18T01:34:18.295+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations creada con exito en DB
[2024-03-18T01:34:19.389+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations_status creada con exito en DB
[2024-03-18T01:34:19.392+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:34:19.391+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:34:23.455+0000] {logging_mixin.py:154} INFO - Datos cargados exitosamente en la base de datos
[2024-03-18T01:34:23.456+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:34:23.456+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:34:25.016+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:34:25.015+0000] {timeout.py:68} ERROR - Process timed out, PID: 192
[2024-03-18T01:34:25.313+0000] {logging_mixin.py:154} INFO - Error al insertar el registro 1: DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 192
[2024-03-18T01:34:55.323+0000] {processor.py:157} INFO - Started process (PID=196) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:34:55.324+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:34:55.325+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:34:55.325+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:35:22.416+0000] {logging_mixin.py:154} INFO - Conectado a Redshift con éxito!
[2024-03-18T01:35:23.217+0000] {logging_mixin.py:154} INFO - Tabla agencies creada con exito en DB
[2024-03-18T01:35:23.996+0000] {logging_mixin.py:154} INFO - Tabla bus_positions creada con exito en DB
[2024-03-18T01:35:25.715+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:35:25.715+0000] {timeout.py:68} ERROR - Process timed out, PID: 196
[2024-03-18T01:35:25.716+0000] {logging_mixin.py:154} INFO - No es posible crear tabla ecobici_stations en DB: 
[2024-03-18T01:35:25.717+0000] {logging_mixin.py:154} INFO - DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 196
[2024-03-18T01:35:26.456+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations_status creada con exito en DB
[2024-03-18T01:35:26.458+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:35:26.457+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:35:29.151+0000] {logging_mixin.py:154} INFO - Datos cargados exitosamente en la base de datos
[2024-03-18T01:35:29.152+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:35:29.152+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:35:55.741+0000] {processor.py:157} INFO - Started process (PID=200) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:35:55.742+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:35:55.743+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:35:55.743+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:36:11.080+0000] {logging_mixin.py:154} INFO - Conectado a Redshift con éxito!
[2024-03-18T01:36:12.062+0000] {logging_mixin.py:154} INFO - Tabla agencies creada con exito en DB
[2024-03-18T01:36:12.919+0000] {logging_mixin.py:154} INFO - Tabla bus_positions creada con exito en DB
[2024-03-18T01:36:13.783+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations creada con exito en DB
[2024-03-18T01:36:14.594+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations_status creada con exito en DB
[2024-03-18T01:36:14.595+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:36:14.595+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:36:17.548+0000] {logging_mixin.py:154} INFO - Datos cargados exitosamente en la base de datos
[2024-03-18T01:36:17.549+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:36:17.549+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:36:25.889+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:36:25.887+0000] {timeout.py:68} ERROR - Process timed out, PID: 200
[2024-03-18T01:36:26.116+0000] {logging_mixin.py:154} INFO - Error al insertar el registro 8: DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 200
[2024-03-18T01:36:56.211+0000] {processor.py:157} INFO - Started process (PID=204) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:36:56.212+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:36:56.213+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:36:56.213+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:37:14.307+0000] {logging_mixin.py:154} INFO - Conectado a Redshift con éxito!
[2024-03-18T01:37:15.998+0000] {logging_mixin.py:154} INFO - Tabla agencies creada con exito en DB
[2024-03-18T01:37:17.017+0000] {logging_mixin.py:154} INFO - Tabla bus_positions creada con exito en DB
[2024-03-18T01:37:18.083+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations creada con exito en DB
[2024-03-18T01:37:20.154+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations_status creada con exito en DB
[2024-03-18T01:37:20.156+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:37:20.156+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:37:22.702+0000] {logging_mixin.py:154} INFO - Datos cargados exitosamente en la base de datos
[2024-03-18T01:37:22.703+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:37:22.703+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:37:26.791+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:37:26.789+0000] {timeout.py:68} ERROR - Process timed out, PID: 204
[2024-03-18T01:37:27.059+0000] {logging_mixin.py:154} INFO - Error al insertar el registro 3: DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 204
[2024-03-18T01:37:56.646+0000] {processor.py:157} INFO - Started process (PID=208) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:37:56.647+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:37:56.649+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:37:56.648+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:38:16.139+0000] {logging_mixin.py:154} INFO - Conectado a Redshift con éxito!
[2024-03-18T01:38:16.815+0000] {logging_mixin.py:154} INFO - Tabla agencies creada con exito en DB
[2024-03-18T01:38:17.738+0000] {logging_mixin.py:154} INFO - Tabla bus_positions creada con exito en DB
[2024-03-18T01:38:18.450+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations creada con exito en DB
[2024-03-18T01:38:19.237+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations_status creada con exito en DB
[2024-03-18T01:38:19.238+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:38:19.238+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:38:22.023+0000] {logging_mixin.py:154} INFO - Datos cargados exitosamente en la base de datos
[2024-03-18T01:38:22.024+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:38:22.024+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:38:26.861+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:38:26.860+0000] {timeout.py:68} ERROR - Process timed out, PID: 208
[2024-03-18T01:38:27.110+0000] {logging_mixin.py:154} INFO - Error al insertar el registro 5: DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 208
[2024-03-18T01:38:57.087+0000] {processor.py:157} INFO - Started process (PID=212) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:38:57.088+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:38:57.090+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:38:57.090+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:39:22.901+0000] {logging_mixin.py:154} INFO - Conectado a Redshift con éxito!
[2024-03-18T01:39:23.952+0000] {logging_mixin.py:154} INFO - Tabla agencies creada con exito en DB
[2024-03-18T01:39:25.051+0000] {logging_mixin.py:154} INFO - Tabla bus_positions creada con exito en DB
[2024-03-18T01:39:26.640+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations creada con exito en DB
[2024-03-18T01:39:27.771+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:39:27.770+0000] {timeout.py:68} ERROR - Process timed out, PID: 212
[2024-03-18T01:39:28.194+0000] {logging_mixin.py:154} INFO - No es posible crear tabla ecobici_stations_status en DB: 
[2024-03-18T01:39:28.195+0000] {logging_mixin.py:154} INFO - DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 212
[2024-03-18T01:39:28.196+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:39:28.196+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:39:31.796+0000] {logging_mixin.py:154} INFO - Datos cargados exitosamente en la base de datos
[2024-03-18T01:39:31.797+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:39:31.797+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:40:28.591+0000] {processor.py:157} INFO - Started process (PID=174) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:40:28.592+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:40:28.594+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:40:28.594+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:40:58.610+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:40:58.608+0000] {timeout.py:68} ERROR - Process timed out, PID: 174
[2024-03-18T01:40:58.620+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:40:58.610+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_daily.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_daily.py", line 4, in <module>
    from scripts.main import data_ingestion
  File "/opt/airflow/scripts/main.py", line 423, in <module>
    data_ingestion()
  File "/opt/airflow/scripts/main.py", line 415, in data_ingestion
    df_ecobici_stations, df_ecobici_stations_status = get_ecobici_data(base_url, params)
  File "/opt/airflow/scripts/main.py", line 253, in get_ecobici_data
    r_ecobiciSS = requests.get(full_url_ecobiciSS, params=params)
  File "/home/airflow/.local/lib/python3.8/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/requests/adapters.py", line 486, in send
    resp = conn.urlopen(
  File "/home/airflow/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 715, in urlopen
    httplib_response = self._make_request(
  File "/home/airflow/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 404, in _make_request
    self._validate_conn(conn)
  File "/home/airflow/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1058, in _validate_conn
    conn.connect()
  File "/home/airflow/.local/lib/python3.8/site-packages/urllib3/connection.py", line 419, in connect
    self.sock = ssl_wrap_socket(
  File "/home/airflow/.local/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 449, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/home/airflow/.local/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 493, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/usr/local/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/usr/local/lib/python3.8/ssl.py", line 1073, in _create
    self.do_handshake()
  File "/usr/local/lib/python3.8/ssl.py", line 1342, in do_handshake
    self._sslobj.do_handshake()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/timeout.py", line 69, in handle_timeout
    raise AirflowTaskTimeout(self.error_message)
airflow.exceptions.AirflowTaskTimeout: DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 174
[2024-03-18T01:40:58.624+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:40:58.646+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/data_ingestion_daily.py took 30.058 seconds
[2024-03-18T01:41:08.769+0000] {processor.py:157} INFO - Started process (PID=180) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:41:08.771+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:41:08.772+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:41:08.772+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:41:28.652+0000] {logging_mixin.py:154} INFO - Conectado a Redshift con éxito!
[2024-03-18T01:41:30.935+0000] {logging_mixin.py:154} INFO - Tabla agencies creada con exito en DB
[2024-03-18T01:41:32.453+0000] {logging_mixin.py:154} INFO - Tabla bus_positions creada con exito en DB
[2024-03-18T01:41:33.854+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations creada con exito en DB
[2024-03-18T01:41:36.100+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations_status creada con exito en DB
[2024-03-18T01:41:36.101+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:41:36.101+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:41:39.462+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:41:39.461+0000] {timeout.py:68} ERROR - Process timed out, PID: 180
[2024-03-18T01:41:39.871+0000] {logging_mixin.py:154} INFO - Error al insertar el registro 2: DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 180
[2024-03-18T01:41:39.872+0000] {logging_mixin.py:154} INFO - Datos cargados exitosamente en la base de datos
[2024-03-18T01:41:39.875+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:41:39.874+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:45:00.952+0000] {processor.py:157} INFO - Started process (PID=174) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:45:00.953+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:45:00.954+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:45:00.954+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:45:19.385+0000] {logging_mixin.py:154} INFO - Conectado a Redshift con éxito!
[2024-03-18T01:45:21.228+0000] {logging_mixin.py:154} INFO - Tabla agencies creada con exito en DB
[2024-03-18T01:45:22.955+0000] {logging_mixin.py:154} INFO - Tabla bus_positions creada con exito en DB
[2024-03-18T01:45:23.909+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations creada con exito en DB
[2024-03-18T01:45:24.925+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations_status creada con exito en DB
[2024-03-18T01:45:24.926+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:45:24.926+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:45:31.144+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:45:31.144+0000] {timeout.py:68} ERROR - Process timed out, PID: 174
[2024-03-18T01:45:31.145+0000] {logging_mixin.py:154} INFO - Error al insertar el registro 2: DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 174
[2024-03-18T01:45:31.146+0000] {logging_mixin.py:154} INFO - Datos cargados exitosamente en la base de datos
[2024-03-18T01:45:31.147+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:45:31.146+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:46:01.381+0000] {processor.py:157} INFO - Started process (PID=178) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:46:01.382+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:46:01.383+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:46:01.383+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:46:19.801+0000] {logging_mixin.py:154} INFO - Conectado a Redshift con éxito!
[2024-03-18T01:46:21.085+0000] {logging_mixin.py:154} INFO - Tabla agencies creada con exito en DB
[2024-03-18T01:46:23.234+0000] {logging_mixin.py:154} INFO - Tabla bus_positions creada con exito en DB
[2024-03-18T01:46:24.185+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations creada con exito en DB
[2024-03-18T01:46:25.685+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations_status creada con exito en DB
[2024-03-18T01:46:25.686+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:46:25.686+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:46:29.165+0000] {logging_mixin.py:154} INFO - Datos cargados exitosamente en la base de datos
[2024-03-18T01:46:29.166+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:46:29.166+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:46:31.747+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:46:31.746+0000] {timeout.py:68} ERROR - Process timed out, PID: 178
[2024-03-18T01:46:31.985+0000] {logging_mixin.py:154} INFO - Error al insertar el registro 1: DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 178
[2024-03-18T01:47:01.809+0000] {processor.py:157} INFO - Started process (PID=182) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:47:01.810+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:47:01.811+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:47:01.811+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:47:21.031+0000] {logging_mixin.py:154} INFO - Conectado a Redshift con éxito!
[2024-03-18T01:47:21.968+0000] {logging_mixin.py:154} INFO - Tabla agencies creada con exito en DB
[2024-03-18T01:47:23.045+0000] {logging_mixin.py:154} INFO - Tabla bus_positions creada con exito en DB
[2024-03-18T01:47:23.870+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations creada con exito en DB
[2024-03-18T01:47:24.553+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations_status creada con exito en DB
[2024-03-18T01:47:24.554+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:47:24.554+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:47:27.181+0000] {logging_mixin.py:154} INFO - Datos cargados exitosamente en la base de datos
[2024-03-18T01:47:27.183+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:47:27.183+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:47:32.267+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:47:32.266+0000] {timeout.py:68} ERROR - Process timed out, PID: 182
[2024-03-18T01:47:32.476+0000] {logging_mixin.py:154} INFO - Error al insertar el registro 5: DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 182
[2024-03-18T01:48:02.249+0000] {processor.py:157} INFO - Started process (PID=186) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:48:02.250+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:48:02.251+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:48:02.251+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:48:24.848+0000] {logging_mixin.py:154} INFO - Conectado a Redshift con éxito!
[2024-03-18T01:48:25.818+0000] {logging_mixin.py:154} INFO - Tabla agencies creada con exito en DB
[2024-03-18T01:48:26.728+0000] {logging_mixin.py:154} INFO - Tabla bus_positions creada con exito en DB
[2024-03-18T01:48:27.671+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations creada con exito en DB
[2024-03-18T01:48:29.167+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations_status creada con exito en DB
[2024-03-18T01:48:29.168+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:48:29.168+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:48:31.543+0000] {logging_mixin.py:154} INFO - Datos cargados exitosamente en la base de datos
[2024-03-18T01:48:31.544+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:48:31.544+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:48:32.310+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:48:32.309+0000] {timeout.py:68} ERROR - Process timed out, PID: 186
[2024-03-18T01:48:32.311+0000] {logging_mixin.py:154} INFO - Error al insertar el registro 0: DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 186
[2024-03-18T01:49:02.704+0000] {processor.py:157} INFO - Started process (PID=190) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:49:02.705+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:49:02.707+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:49:02.706+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:49:15.507+0000] {logging_mixin.py:154} INFO - Conectado a Redshift con éxito!
[2024-03-18T01:49:16.337+0000] {logging_mixin.py:154} INFO - Tabla agencies creada con exito en DB
[2024-03-18T01:49:17.118+0000] {logging_mixin.py:154} INFO - Tabla bus_positions creada con exito en DB
[2024-03-18T01:49:17.948+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations creada con exito en DB
[2024-03-18T01:49:18.828+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations_status creada con exito en DB
[2024-03-18T01:49:18.829+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:49:18.829+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:49:21.903+0000] {logging_mixin.py:154} INFO - Datos cargados exitosamente en la base de datos
[2024-03-18T01:49:21.904+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:49:21.903+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:49:33.103+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:49:33.102+0000] {timeout.py:68} ERROR - Process timed out, PID: 190
[2024-03-18T01:49:33.104+0000] {logging_mixin.py:154} INFO - Error al insertar el registro 9: DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 190
[2024-03-18T01:50:03.181+0000] {processor.py:157} INFO - Started process (PID=194) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:50:03.182+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:50:03.183+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:50:03.183+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:50:18.637+0000] {logging_mixin.py:154} INFO - Conectado a Redshift con éxito!
[2024-03-18T01:50:20.027+0000] {logging_mixin.py:154} INFO - Tabla agencies creada con exito en DB
[2024-03-18T01:50:20.965+0000] {logging_mixin.py:154} INFO - Tabla bus_positions creada con exito en DB
[2024-03-18T01:50:22.029+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations creada con exito en DB
[2024-03-18T01:50:22.925+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations_status creada con exito en DB
[2024-03-18T01:50:22.926+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:50:22.926+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:50:26.164+0000] {logging_mixin.py:154} INFO - Datos cargados exitosamente en la base de datos
[2024-03-18T01:50:26.165+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:50:26.164+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:50:33.837+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:50:33.835+0000] {timeout.py:68} ERROR - Process timed out, PID: 194
[2024-03-18T01:50:34.223+0000] {logging_mixin.py:154} INFO - Error al insertar el registro 6: DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 194
[2024-03-18T01:51:03.623+0000] {processor.py:157} INFO - Started process (PID=198) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:51:03.624+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:51:03.625+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:51:03.625+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:51:19.359+0000] {logging_mixin.py:154} INFO - Conectado a Redshift con éxito!
[2024-03-18T01:51:20.350+0000] {logging_mixin.py:154} INFO - Tabla agencies creada con exito en DB
[2024-03-18T01:51:21.190+0000] {logging_mixin.py:154} INFO - Tabla bus_positions creada con exito en DB
[2024-03-18T01:51:22.150+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations creada con exito en DB
[2024-03-18T01:51:23.712+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations_status creada con exito en DB
[2024-03-18T01:51:23.713+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:51:23.713+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:51:29.837+0000] {logging_mixin.py:154} INFO - Datos cargados exitosamente en la base de datos
[2024-03-18T01:51:29.839+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:51:29.839+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:51:33.976+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:51:33.976+0000] {timeout.py:68} ERROR - Process timed out, PID: 198
[2024-03-18T01:51:34.346+0000] {logging_mixin.py:154} INFO - Error al insertar el registro 3: DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 198
[2024-03-18T01:52:04.065+0000] {processor.py:157} INFO - Started process (PID=202) to work on /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:52:04.066+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/data_ingestion_daily.py for tasks to queue
[2024-03-18T01:52:04.067+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:52:04.067+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_daily.py
[2024-03-18T01:52:23.021+0000] {logging_mixin.py:154} INFO - Conectado a Redshift con éxito!
[2024-03-18T01:52:24.071+0000] {logging_mixin.py:154} INFO - Tabla agencies creada con exito en DB
[2024-03-18T01:52:25.990+0000] {logging_mixin.py:154} INFO - Tabla bus_positions creada con exito en DB
[2024-03-18T01:52:27.731+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations creada con exito en DB
[2024-03-18T01:52:30.304+0000] {logging_mixin.py:154} INFO - Tabla ecobici_stations_status creada con exito en DB
[2024-03-18T01:52:30.305+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:52:30.304+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
[2024-03-18T01:52:34.332+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:52:34.331+0000] {timeout.py:68} ERROR - Process timed out, PID: 202
[2024-03-18T01:52:34.333+0000] {logging_mixin.py:154} INFO - Error al insertar el registro 2: DagBag import timeout for /opt/airflow/dags/data_ingestion_daily.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.7.3/best-practices.html#reducing-dag-complexity, PID: 202
[2024-03-18T01:52:34.334+0000] {logging_mixin.py:154} INFO - Datos cargados exitosamente en la base de datos
[2024-03-18T01:52:34.335+0000] {logging_mixin.py:154} INFO - [2024-03-18T01:52:34.335+0000] {utils.py:56} INFO - Cargando datos en la base de datos...
